### DISCUSSION TITLE
"Enable a faster Labs boot"?

### DISCUSSION THREAD

**Discussion by jamesdeluk** [gopro/labs#1012] [Q&A] [answered]
For USB Power Trigger (https://gopro.github.io/labs/control/usb/), what does the "Enable a faster Labs boot." checkbox do?

**Comment by dnewman-gpsw** [comment, marked as answer]
It enables a faster boot.  All *FAST=1 does is remove the splash messages, like the owner message, or beta message and the features activated list.

**Reply by jamesdeluk** [reply]
Ah, thanks!
